source/types.ts

```diff
@@ -161,8 +161,27 @@ export interface MutationOptions {
   decodeDatesAsIso?: boolean;
 }
 
+type SimpleTypeSchemaProperty = {
+  type: "string" | "boolean" | "number" | "array";
```

Comment:
There are a few more types available that are not represented on the support page you were referencing, also "array" accidentally snuck in there:
```suggestion
  type: "string" | "boolean" | "number" | "integer" | "variable";
```

Reply:
Thanks! Silly me - didn't see that. Nicely spotted!